src/Intercom/Data/AdminConversationReply.cs

```diff
@@ -13,7 +13,7 @@ namespace Intercom.Data
 {
     public class AdminConversationReply : Reply
     {
-        public string type
+        public override string type
```

Comment:
This is my own dotnet stupidity but why do we need an override here now and not before?

Reply:
We needed this before, just didn't catch it. 🙈